Bump commons-ui 147

flomillot opened the pull request:
Rename "enableDeveloperMode" to "isDeveloperMode" across the codebase
Also include https://github.com/gridsuite/gridexplore-app/pull/783

EtienneLt committed:
block edit equipment type for filter outside dev mode

Signed-off-by: Etienne LESOT <etienne.lesot@rte-france.com>

EtienneLt committed:
revert

Signed-off-by: Etienne LESOT <etienne.lesot@rte-france.com>

EtienneLt committed:
fix

Signed-off-by: Etienne LESOT <etienne.lesot@rte-france.com>

flomillot committed:
Rename "enableDeveloperMode" to "isDeveloperMode" across the codebase

Standardizes variable naming to enhance readability and align with boolean naming conventions.

Signed-off-by: Florent MILLOT <75525996+flomillot@users.noreply.github.com>

EtienneLt committed:
Merge branch 'main' into block-edit-equipment-type-for-filter

EtienneLt commented on src/redux/actions.ts:
```diff
@@ -60,14 +60,14 @@ export function selectComputedLanguage(computedLanguage: AppState['computedLangu
-    enableDeveloperMode: boolean;
+    isDeveloperMode: boolean;
 };
 export function selectEnableDeveloperMode(
```
```suggestion
export function selectIsDeveloperMode(
```

EtienneLt replied:
remove blank

EtienneLt commented:
should 
```java
const [enableDeveloperModeLocal, handleChangeDeveloperMode] = useParameterState(PARAM_DEVELOPER_MODE);
```
be changed ?

flomillot committed:
Rename developer mode state variable for clarity.

Renamed `enableDeveloperModeLocal` to `isDeveloperMode` in both `app-top-bar.jsx` and `app-top-bar.tsx` components. This improves readability and aligns variable naming with boolean conventions.

Signed-off-by: Florent MILLOT <75525996+flomillot@users.noreply.github.com>

flomillot commented:
> should
> 
> ```java
> const [enableDeveloperModeLocal, handleChangeDeveloperMode] = useParameterState(PARAM_DEVELOPER_MODE);
> ```
> 
> be changed ?

yes I forgot thank you

flomillot committed:
prettier

Signed-off-by: Florent MILLOT <florent.millot_externe@rte-france.com>

EtienneLt reviewed (approved):
Code OK

EtienneLt reviewed (approved):
code OK

flomillot committed:
Merge remote-tracking branch 'origin/block-edit-equipment-type-for-filter' into rename-developer-mode-var

# Conflicts:
#	src/components/directory-content-dialog.tsx

flomillot committed:
Upgrade @gridsuite/commons-ui to v0.147.0

Signed-off-by: Florent MILLOT <75525996+flomillot@users.noreply.github.com>
